source/modules/MEI.xml

```diff
@@ -510,6 +510,25 @@
       </valList>
     </content>
   </macroSpec>
+  <macroSpec ident="data.CANCELACCID" module="MEI" type="dt">
+    <desc xml:lang="en">Indicates the location of the tonic in the circle of fifths.</desc>
+    <content>
+      <valList type="closed">
+        <valItem ident="none">
+          <desc xml:lang="en">Do not show cancellation accidentals.</desc>
+        </valItem>
+        <valItem ident="before">
+          <desc xml:lang="en">Show cancellation accidentals before the new key accidentals.</desc>
+        </valItem>
+        <valItem ident="after">
+          <desc xml:lang="en">Show cancellation accidentals after the new key accidentals ("Old style" or "French")</desc>
+        </valItem>
+        <valItem ident="before-bar">
```

Comment:
I don't think this value makes much sense here, as `keySig` is a layer element and does not have do appear next to a measure.

Random example from Ravel's _Le tombeau de Couperin_

<img width="250" alt="Le tombeau de Couperin (excerpt)" src="https://github.com/music-encoding/music-encoding/assets/7693447/f6bb2a07-98fd-4d7b-9bd3-ee50c9b32904">

If you need to split it up, you should encode the cancellation at the end of one measure and the new one at the beginning of the next. (Maybe connect them with a linking attribute like `@precedes` and `@follows`). 

Or what to do if you have a `barLine` element next to it, but in the following order: 

```xml
<note />
<keySig keysig.cancelaccid="before-bar" keysig="3f" />
<barLine />
<note />
```

Reply:
(Sorry, I had written this response before I resolved it, but it didn't save for some reason.)

> Random example from Ravel's Le tombeau de Couperin

You can still do this:

`<keySig keysig.cancelaccid="after" ... />`

We make no guarantees in MEI that all values are valid in all circumstances.